lib/perl/Genome/Model/Tools/DetectVariants2/SpeedseqSv.pm

```diff
@@ -0,0 +1,132 @@
+#SpeedseqSv.pm
+
+
+package Genome::Model::Tools::DetectVariants2::SpeedseqSv;
+use warnings;
+use strict;
+
+use Genome;
+use File::Basename;
+use IPC::System::Simple;
+use Genome::File::Tsv;
```

Comment:
Check and see whether all of these packages are actually used, and remove ones that aren't